gdpr/gdprUITests/gdprUITests.m

```diff
@@ -0,0 +1,40 @@
+//
```

Comment:
I prefer to add the entire opt out flow in a Test so it makes our testing in future releases easier. If we feel like it's not needed lets remove the test target and associated files